EIPS/pay_req_url_fmt.md

```diff
@@ -0,0 +1,71 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: URL Format for Payment Requests
+    Author: Daniel A. Nagy <daniel@ethereum.org>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2017-08-01
+    Requires: #20, #137
+
+## Simple Summary
+A standard way of representing payment requests in Ethers and ERC #20 tokens as URLs.
+
+## Abstract
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very 
+loosely coupled applications. A standardized URL format for payment requests allows for instant invocation of the user's preferred 
+wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the payment transaction only to be 
+confirmed by the (authenticated) user.
```

Comment:
Style: Recommend not including hard-line breaks in prose text.  As is, it is a bit of a pain to read on a 1900px wide window (split diff), and similarly hard to read on a phone.